lib/core/src/display/rendering/dom_container.rs

```diff
@@ -47,36 +60,76 @@ impl DOMContainerData {
 /// A collection for holding 3D `Object`s.
 #[derive(Debug)]
 pub struct DOMContainer {
-    pub dom          : HtmlElement,
-    resize_observer  : Option<ResizeObserver>,
-    data             : Rc<RefCell<DOMContainerData>>,
+    pub dom               : HtmlElement,
+    resize_observer       : Option<ResizeObserver>,
+    intersection_observer : Option<IntersectionObserver>,
```

Comment:
what is "intersection" observer. I mean, I understand it is this : https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API

But what we use it for here? The answer may be abvious but I dont see it yet. In any case, both the information why we use intersection observer as well as the resize observer should be in the docs.

Reply:
It's used to get the DOM's Rect. I am using the DOM's position in `navigator.rs:l90` to calculate its center.